suites/quincy/rados/tier-2_rados_mero_4_node_add_db_wal_using_bst.yaml

```diff
@@ -0,0 +1,157 @@
+# Suite contains  tier-2 rados tests which adds DB and wal to the OSD
+#===============================================================================================
+#------------------------------------------------------------------------------------------
+#-----      Tier-2- Using bluestore-tool add DB & Wal  to the existing OSD          ------
+#------------------------------------------------------------------------------------------
+# Conf: conf/quincy/baremetal_pipeline/mero_4_node_4_client_conf.yaml
+#
```

Comment:
Please add the comment stating the requirements/ limitations of the test suite.

Reply:
Added the requirement details.